server/verifiers/pom.xml

```diff
@@ -35,5 +35,6 @@
     </properties>
     <modules>
         <module>trustful-verifier</module>
+        <module>facebook-verifier</module>
```

Comment:
add facebook-verifier implementation to operations server dependencies and add corresponding configuration module to admin ui dependencies